packages/manager/src/features/TopMenu/CreateMenu/CreateMenu.styles.ts

```diff
@@ -7,42 +7,47 @@ import Add from 'src/assets/icons/add.svg';
 export const StyledHeading = styled('h3', {
   label: 'StyledHeading',
   shouldForwardProp: omittedProps(['paddingTop']),
-})<{ paddingTop?: boolean }>(({ theme, ...props }) => ({
+})<{ marginTop?: boolean }>(({ theme, ...props }) => ({
   '& svg': {
     height: 20,
     marginRight: theme.spacing(1),
     width: 20,
   },
   alignItems: 'center',
-  color: theme.name === 'dark' ? theme.tokens.color.Neutrals[50] : 'inherit',
+  color: theme.tokens.content.Text.Secondary.Default,
   display: 'flex',
-  font: theme.font.bold,
-  fontSize: '0.7rem',
+  fontSize: theme.tokens.font.FontSize.Xxxs,
+  // eslint-disable-next-line @linode/cloud-manager/no-custom-fontWeight
+  fontWeight: theme.tokens.font.FontWeight.Extrabold,
   letterSpacing: '1px',
+  lineHeight: theme.tokens.font.LineHeight.Xxxs,
   margin: 0,
   padding: '8px 14px',
-  textTransform: 'uppercase',
+  textTransform: theme.tokens.font.Textcase.Uppercase,
   [theme.breakpoints.up('lg')]: {
     background: 'inherit',
```

Comment:
The headers for Storage and Database are still using the older color when in dark mode & md/sm breakpoints:

https://github.com/user-attachments/assets/f3d01c24-60f0-4cbf-9d75-7c41f38c8605

I can't seem to figure out why only these two headers are affected but applying the following property seems to fix it:


```suggestion
  backgroundColor: theme.tokens.background.Normal,
  [theme.breakpoints.up('lg')]: {
```